spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -82,39 +83,42 @@
 import com.rabbitmq.client.Envelope;
 
 /**
- * A {@link org.springframework.cloud.stream.binder.Binder} implementation backed by RabbitMQ.
+ * A {@link org.springframework.cloud.stream.binder.Binder} implementation backed by
+ * RabbitMQ.
+ *
  * @author Mark Fisher
  * @author Gary Russell
  * @author Jennifer Hickey
  * @author Gunnar Hillert
  * @author Ilayaperumal Gopinathan
  * @author David Turanski
  * @author Marius Bogoevici
+ * @author Artem Bilan
  */
 public class RabbitMessageChannelBinder
-		extends AbstractMessageChannelBinder<ExtendedConsumerProperties<RabbitConsumerProperties>,
-		ExtendedProducerProperties<RabbitProducerProperties>, RabbitExchangeQueueProvisioner>
-		implements ExtendedPropertiesBinder<MessageChannel, RabbitConsumerProperties, RabbitProducerProperties> {
+		extends AbstractMessageChannelBinder<ExtendedConsumerProperties<RabbitConsumerProperties>, ExtendedProducerProperties<RabbitProducerProperties>, RabbitExchangeQueueProvisioner>
```

Comment:
Ugly long line

Reply:
Ugly Eclipse formatter plugin in IDEA...
I will try to reinstate it though.
Thank you for the pointer!